tests/python/PyUnusedCodeBearTest.py

```diff
@@ -44,6 +57,12 @@
                                ],
                 settings={'remove_all_unused_imports': True})
 
+PyUnusedVariableTest = verify_local_bear(
+                PyUnusedCodeBear,
+                valid_files=[valid_code],
+                invalid_files=[invalid_code],
+                settings={'remove_unused_variables': True})
```

Comment:
Try to also test if `removed_unused_variables` is `False`. You can omit the `invalid_files` for that.